config/console.php

```diff
@@ -2,5 +2,11 @@
 
 declare(strict_types=1);
 
+use Yiisoft\Aliases\Aliases;
+
 return [
+    /** ALIASES */
```

Comment:
It's clear that it is aliases without this comment.

Reply:
fixed.